scripts/duplicate_end_to_end.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This file runs the duplicate model end-to-end,
+# Prerequisites:
+# 1. Train a similarity model to get bugs similar to a given bug
+# 2. Train the duplicate model independently
+# During testing, we first generate all bugs similar to a bug using model 1,
+# then apply the trained duplicate model to this set of similar bugs and classify
+# the bug as duplicate if the probability score is greater than a threshold. (0.8)
+
+# The training steps are skipped here; this can already be done through similarity_trainer.py
+# and trainer.py. Hence, **similaritymodel** and **duplicatemodel** pretrained files
+# are already available.
+
+import argparse
+import csv
+import os
+import sys
+from datetime import datetime, timedelta
+from logging import INFO, basicConfig, getLogger
+
+import requests
+
+from bugbug import bugzilla, similarity
+from bugbug.models.duplicate import DuplicateModel
+from bugbug.utils import download_check_etag, zstd_decompress
+
+basicConfig(level=INFO)
+logger = getLogger(__name__)
+
+URL = "https://community-tc.services.mozilla.com/api/index/v1/task/project.relman.bugbug.train_similarity.latest/artifacts/public/{}.zst"
+
+REPORTERS_TO_IGNORE = {"intermittent-bug-filer@mozilla.bugs", "wptsync@mozilla.bugs"}
+
+
+def parse_args(args):
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "similaritymodel",
+        help="Similarity algorithm to test",
+        choices=similarity.model_name_to_class.keys(),
+    )
+
+    # At the moment, we use LinearSVC
+    parser.add_argument("duplicatemodel", help="Duplicate algorithm to test")
```

Comment:
I'd remove this argument, and just load the duplicate model from the file, whatever the underlying algorithm of the model is